site/layouts/partials/navbar.html

```diff
@@ -0,0 +1,29 @@
+<nav class="navbar navbar-expand-md navbar-light bg-light">
```

Comment:
The navbar partial is currently unused. I also don't see it when rendering locally.

Reply:
Your browser is probably caching. I see whatever is on the staging site locally.

Reply:
Hmm.. so I'm talking about the navbar partial. All other partials have usages such as `{{ partial "header.html" . }}`. The navbar doesn't. Maybe I don't understand how this whole thing works?

I also reloaded via Ctrl+F5 to force reloading of cached content. It's still the same. But the logo is gone now. So I suspect something is going on with the logo as well.

Reply:
My hugo version is `Hugo Static Site Generator v0.60.0-DEV/extended`. Is that too old?

Reply:
Hmm yeah, worth getting v0.63.1

Reply:
So after seriously doubting my sanity, I think `header.html` is broken:
```
<a href="{{"/" | relURL }}"><img src="{{ "/krew-logo.svg" }}" height="100vw"/></a>
```
should be
```
<a href="{{"/" | relURL }}"><img src="{{ "/img/krew-logo.svg" }}" height="100vw"/></a>
```

And most likely the navbar partial needs to be included there as well.